portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase

```diff
@@ -50,6 +50,73 @@ definition {
 			vocabularyName = "Vocabulary Name");
 	}
 
+	@description = "This is a use case for LPS-107730. Can not store 256 character in name field, key field and value field of category."
+	@priority = "4"
+	test CannotStore256CharactersInNameFieldKeyFieldAndValueField {
+		var actualText = "The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256";
+		var inputText = "The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.";
+		var modifiedText = "The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255";
```

Comment:
It has 256 characters, is it right?

Reply:
The value of modified text has 255 characters.